lib/gemsurance/yml_formatter.rb

```diff
@@ -0,0 +1,11 @@
+module Gemsurance
+  class YmlFormatter
+    def initialize(gem_infos)
+      @gem_infos = gem_infos
+    end
+
+    def format
+      ERB.new(File.read(File.join(File.dirname(File.expand_path(__FILE__)), 'templates/output.yml.erb')), nil, '-').result(binding)
```

Comment:
I've just copied current `html_formatter.rb` code, but you're totally right, I'll create a cleaner `BaseFormatter`class and use it in `HtmlFormatter` and `YmlFormatter`